### ISSUE TITLE
Edit My Profile, fields "PRIVACY" not open all options

### ISSUE THREAD

**Issue by GabryNet** [SocialEngine/phpv4-issues#713]
(Thanks for reporting an issue! Please make sure you click the link above to view the issue guidelines, then fill out the blanks below.)

What are the steps to reproduce this issue?
-------------------------------------------
1. …Go to edit my profile
2. …Click on the "world globe icon"
3. …Should open the options to choose who can see that field

What happens?
-------------
…Is showing only the first option "Everyone", the rest of them are hidden. This was tested also on SE official demo.

What were you expecting to happen?
----------------------------------
…We have to be able to choose who we want to see that info field.
![untitled](https://cloud.githubusercontent.com/assets/25679777/25473593/531fc1dc-2b30-11e7-9a9a-57e0c6b7ddc1.png)

What versions of software are you using?

**SocialEngine PHP Version:** … 4.9

**Comment by DonnaScriptTechs** [comment]
Thank you for reporting this issue. I've added it to our list to fix.

**Comment by adira-g** [comment]
Weird, on my installation (insignia on SE 4.9) it seems to be fine. I wonder if this bug appears only on 4.9.1? (before I upgrade)

I get the issue in SE demo, too. Same browser my site works perfect. Only difference ism I don't use vertical menus.

**Comment by DonnaScriptTechs** [comment]
It could be with 4.9.1 perhaps. I get the same issue on my local installs. Thanks for providing your feedback.

**Comment by JonSimon93** [comment]
I use to hide this, with display none, and just control it globally as admin.
disliked seeing these on signup when i had Social E

**Comment by Elshara** [comment]
Same issue here can reproduce.

**Comment by RaymondBenc** [comment]
Thank you for brining this to our attention. We are looking into get this resolved for our next release, however in the meantime here is a quick fix I tested. To apply the patch...

1. Go to your Admin Control Panel and visit `Appearance -> Theme Editor`
1. Edit `theme.css` for your active theme.
1. At the end of that file, add
```
.global_form, .global_form div.form-wrapper, .global_form > div > div, .global_form > div {
    overflow: visible;
}
```
4. Click `Save Changes`

When the patch is applied, your drop down should show up like this

![screen shot 2017-04-29 at 16 24 36](https://cloud.githubusercontent.com/assets/6339284/25556235/7a0d05e6-2cf9-11e7-8bfa-64bce9400f31.png)

**Disclaimer**: *Like with any quick fix, this could cause issues in other areas as it was not tested by QC.*

If the fix works out for you, let us know.

**Comment by Elshara** [comment]
Thanks for the fix! Looking forward to its implementation in next update.

**Comment by RaymondBenc** [comment]
@DonnaScriptTechs is helping with some tests, I updated the patch to focus the fix to just this specific page so it does not affect other sections.

```
#global_page_user-edit-profile .global_form, 
#global_page_user-edit-profile .global_form div.form-wrapper, 
#global_page_user-edit-profile .global_form > div > div, 
#global_page_user-edit-profile .global_form > div {
    overflow: visible;
}
```

**Comment by Elshara** [comment]
Awesome! She is quite the welcome addition around here. AS are you.

**Comment by GabryNet** [comment]
Thank you for the fast fix , work's perfect ... have a good day !

**Comment by Elshara** [comment]
:)

**Comment by DonnaScriptTechs** [comment]
This is fixed for 4.9.2 when it is released. If you still have this issue after we release, please open a new bug report.
